ChangeLog

```diff
@@ -1,3 +1,9 @@
+2024-05-23 Andrew Johnson <andrew.johnson@arjohnsonau.com>
+        
+        * Added variadic templates to be used instead of the generated code
```

Comment:
This usually starts with a 'where', commonly a single file.  Mayne add `inst/include/Rcpp/*: ` as a wildcard?  With or without the star?

Dont't sweat over the details, I can clean this up later I just thought I'd point it out before your next PR :stuck_out_tongue_winking_eye:

Reply:
Good to know! 

That's the last of my changes/updates, so feel free to let me know if there are any other things to cleanup here - otherwise I'll wait to hear if I managed to break anything in the reverse-dependency checks!

Reply:
Yep.  Because the (large, careful) PR came in stages and because the rev.dep machine runs slowwwwwwwwwly we are still at stage 2 of 3.  I will re-start with the now complete set of changes so it'll be a few days ... but this is looking very very good, and it is both help -- and appreciated!  So big thank you!